202009.0/fd417f8f-b72a-4d8c-a134-8230ea1a0dbc.md

```diff
@@ -1,16 +1,3 @@
-> Audience:
```

Comment:
It's TW internals.
The audience should be managed by naming, portal, and navigation.
It's not a job of a reader to investigate to what auditory group they belong to.

Reply:
That was my initiative to have this into docker/sdk repo in order to inform readers would it be useful for them or not.

I assume this is not necessary in 360 documentation.